account_customer_wallet/models/account_account.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Coop IT Easy SCRLfs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Account(models.Model):
+    _inherit = "account.account"
+
+    # TODO: Figure out whether there already exists a module that makes this
+    # One2many field available
+    move_line_ids = fields.One2many(
```

Comment:
as ``_compute_customer_wallet_balance`` is stored to False, I think that we can remove the @api.depends in the funciton ``_compute_customer_wallet_balance``. Odoo core account does the same for the field ``total_invoiced``. 
https://github.com/odoo/odoo/blob/12.0/addons/account/models/partner.py#L389

It simplify the algorigthm and make the field ``move_line_ids`` unncessary.

Reply:
@legalsylvain For clarity's sake: a compute function being `stored=False` means that the value will be recomputed every time the field is accessed? I was under the impression that the result is cached internally.

Reply:
Makes sense, I agree. I've implemented this in [0fe9b1a](https://github.com/coopiteasy/addons/pull/222/commits/0fe9b1a8841e4a92776ed46c09ea97de20ccd23e) and [a2ead73](https://github.com/coopiteasy/addons/pull/222/commits/a2ead7390b9479a396abf469ffa6e354375560b4).